docs/pages/management/admin/trustedclusters.mdx

```diff
@@ -1,500 +1,594 @@
 ---
-title: Teleport Trusted Clusters
-description: How to configure access and trust between two Teleport Clusters.
+title: Configure Trusted Clusters
+description: Explains how you can configure a trust relationship and manage access between two Teleport clusters.
 h1: Trusted Clusters
 ---
 
-Teleport can partition compute infrastructure into multiple clusters. A cluster
-is a group of resources connected to a Teleport Auth Server. Each cluster
-manages a separate certificate authority (CA) for all users and resources.
+As you learned in [Core Concepts](../../core-concepts.mdx), a Teleport cluster
+consists of the Teleport Auth Service, the Teleport Proxy Service, and the 
+Teleport services that manage access to resources in your infrastructure.
+With Teleport, you can also partition your infrastructure into multiple connected 
+clusters to allow the users of one cluster—the **root cluster**—to connect to resources 
+in another cluster—the **leaf cluster**—while authenticated using a single 
+Teleport Auth Service. 
 
-Trusted Clusters allow the users of one cluster, the **root cluster**, to
-seamlessly connect to resources in another cluster, the **leaf cluster**, while
-remaining authenticated with only a single Auth Service. The leaf cluster can
-run behind a firewall without any ingress ports open.
+After you establish a trust relationship between the root and leaf clusters, the leaf cluster can
+run behind a firewall without any inbound ports open. The leaf cluster creates an outbound reverse 
+SSH tunnel to the root cluster and keeps the tunnel open.
+When a user tries to connect to any resource in the leaf cluster, the connection goes through the 
+reverse tunnel using the root cluster's proxy.
 
-Uses for Trusted Clusters include:
+## Who uses trusted clusters?
+
+Most organizations don't need to configure trusted clusters. In most cases, you can enroll
+resources and add reverse-tunnel proxies without creating a new cluster. Typically, a single 
+teleport cluster can scale to hundreds of thousands of connected resources.
```

Comment:
Hundreds of thousands is a bit ambitious. How about tens of thousands instead?

Reply:
to be fair, I didn't write that. It came from the architecture page: "In many cases, a single cluster is enough. A single teleport cluster can scale to hundreds of thousands of connected resources!" 
But, yeah, it felt awkward and unnecessary. I'm ditching it in favor of something simpler.